genesis/process/sovereignGenesisBlockCreator.go

```diff
@@ -65,3 +76,204 @@ func (gbc *sovereignGenesisBlockCreator) createSovereignEmptyGenesisBlocks() (ma
 
 	return mapEmptyGenesisBlocks, nil
 }
+
+func (gbc *sovereignGenesisBlockCreator) computeSovereignDNSAddresses(enableEpochsConfig config.EnableEpochs) error {
+	initialAddresses, err := factory.DecodeAddresses(gbc.arg.Core.AddressPubKeyConverter(), gbc.arg.DNSV2Addresses)
+	if err != nil {
+		return err
+	}
+
+	return gbc.computeDNSAddresses(enableEpochsConfig, initialAddresses)
+}
+
+func (gbc *sovereignGenesisBlockCreator) createSovereignHeaders(args *headerCreatorArgs) (map[uint32]data.HeaderHandler, error) {
```

Comment:
We should add some unit tests for these new methods (~5) added.

Reply:
These are already all being tested through existing tests.
Extended them to increase full coverage without "return error" paths

Reply:
This test: `TestSovereignGenesisBlockCreator_CreateGenesisBaseProcess` tests ~75% of the new code